test/llvm_ir_correct/dllimport_test.f90_mod.f90

```diff
@@ -0,0 +1,28 @@
+!
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+!
+! REQUIRES: system-windows
+! RUN: true
```

Comment:
is anything being tested here?

Reply:
ok, I suppose `%flang -S -emit-llvm -S -emit-llvm %s_mod.f90 %s` these things relate to each other. Would be great to comment that - one can't figure it out by looking at the file itself.

Reply:
Added comments to tests. Thanks.